python/sbp/client/drivers/network_drivers.py

```diff
@@ -16,8 +16,6 @@
 # TODO (Buro): Consider making a basic, standard library async tcp
 # client: http://pymotw.com/2/asyncore/.
 
-DEFAULT_RECV_SIZE = 1024
-
 class TCPDriver(BaseDriver):
   """TCPDriver
```

Comment:
Have you tested to see if this TCP driver and serial link can work together for exchanging observations?